libs/sdk-ui-dashboard/src/model/events/userInteraction.ts

```diff
@@ -21,6 +21,34 @@ export type KpiAlertDialogOpenedPayload = UserInteractionPayloadWithDataBase<
     }
 >;
 
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedFrom = "kpi" | "widget" | "insight";
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedType = "inherit" | "custom";
+/**
+ * @alpha
+ */
+export type DescriptionTooltipOpenedData = {
+    from: DescriptionTooltipOpenedFrom;
+    type: DescriptionTooltipOpenedType;
+    descriptionLength: number;
```

Comment:
Do we really want to send description length only here? Maybe sending the ful description will make it more future proof?

Reply:
ok, will provide whole description and use only length later in KD when creating telemetry event